external-crates/move/crates/move-compiler/src/command_line/compiler.rs

```diff
@@ -5,18 +5,18 @@
 use crate::{
     cfgir::{self, visitor::AbsIntVisitorObj},
     command_line::{DEFAULT_OUTPUT_DIR, MOVE_COMPILED_INTERFACES_DIR},
-    compiled_unit,
-    compiled_unit::AnnotatedCompiledUnit,
+    compiled_unit::{self, AnnotatedCompiledUnit},
     diagnostics::{
         codes::{Severity, WarningFilter},
         *,
     },
     editions::Edition,
-    expansion, hlir, interface_generator, naming, parser,
-    parser::{comments::*, *},
```

Comment:
`self` plus `*` seems dubious?

we should really write a style guide for imports and make the code all use it...